### PR TITLE
feat: [CDS-46329]: deployment freeze enhancements

### PR DESCRIPTION
**Pull request by udayvunnam**
### Summary

- Global freeze toggle off from UI when freeze time is over - add temporary solution from UI, until the framework of the event is done on the backend, where the toggle is updated to off once the freeze time is over.
- Bulk select - disable RBAC
- add navigation links to  specific global freeze in the banner

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/72-freeze-windows/components/GlobalFreezeToggle/GlobalFreezeToggle.tsx [hunk 1]
```diff
@@ -63,6 +62,9 @@ export const GlobalFreezeToggle: FC<GlobalFreezeToggleProps> = ({ freezeListLoad
     }
   })
 
+  const isGlobalFreezeEnabled =
+    getGlobalFreezeData?.data?.status === 'Enabled' && !isEmpty(getGlobalFreezeData?.data.currentOrUpcomingWindow)
```
Comment (bhavya-sinha) [inline src/modules/72-freeze-windows/components/GlobalFreezeToggle/GlobalFreezeToggle.tsx:66]: Can we have a proper enum for the status?

**Review by bhavya-sinha — APPROVED** [review]

**Review by ishantmehtaharness — APPROVED** [review]
Looks Good Uday

File: src/modules/72-freeze-windows/components/GlobalFreezeToggle/GlobalFreezeToggle.tsx (hunk 1 above)
Comment (udayvunnam) [reply src/modules/72-freeze-windows/components/GlobalFreezeToggle/GlobalFreezeToggle.tsx:66]: Status is already string union and type-safe for all operations

**Comment by udayvunnam** [comment]
retrigger cypress-cv

**Review by pankajsingh-harness — APPROVED** [review]

**Pull request merged**